### ISSUE TITLE
Jackson shouldn't serialize null

### ISSUE THREAD

**Issue by Riduidel** [Riduidel/aadarchi-technology-detector#75]
Typically, in cached artifact loader files, we have a ton of null values which make files way too big.

Here is as an example an extract of the `.cache/pypi/popular_pypi_artifacts.json`

```
{
  "groupId" : null,
  "artifactId" : null,
  "name" : "psycopg-binary",
  "description" : null,
  "licenses" : null,
  "categories" : null,
  "tags" : null,
  "downloads" : null,
  "ranking" : null,
  "users" : null,
  "previousUsers" : null,
  "interpolatedUsers" : null,
  "infered" : null,
  "repositories" : null,
  "versions" : null,
  "urls" : null,
  "githubDetails" : null,
  "coordinates" : ""
}, {
  "groupId" : null,
  "artifactId" : null,
  "name" : "keras-applications",
  "description" : null,
  "licenses" : null,
  "categories" : null,
  "tags" : null,
  "downloads" : null,
  "ranking" : null,
  "users" : null,
  "previousUsers" : null,
  "interpolatedUsers" : null,
  "infered" : null,
  "repositories" : null,
  "versions" : null,
  "urls" : null,
  "githubDetails" : null,
  "coordinates" : ""
}, {
  "groupId" : null,
  "artifactId" : null,
  "name" : "flask-jwt-extended",
  "description" : null,
  "licenses" : null,
  "categories" : null,
  "tags" : null,
  "downloads" : null,
  "ranking" : null,
  "users" : null,
  "previousUsers" : null,
  "interpolatedUsers" : null,
  "infered" : null,
  "repositories" : null,
  "versions" : null,
  "urls" : null,
  "githubDetails" : null,
  "coordinates" : ""
}, {
  "groupId" : null,
  "artifactId" : null,
  "name" : "bidict",
  "description" : null,
  "licenses" : null,
  "categories" : null,
  "tags" : null,
  "downloads" : null,
  "ranking" : null,
  "users" : null,
  "previousUsers" : null,
  "interpolatedUsers" : null,
  "infered" : null,
  "repositories" : null,
  "versions" : null,
  "urls" : null,
  "githubDetails" : null,
  "coordinates" : ""
}
```

I'm quite sure it's possible to configure Jackson to not serialize null values. But how?

**Comment by Riduidel** [comment]
To test it, that's quite simple:

1. Create an `ArtifactDetails` object
2. Set its name
3. Write it using `FileHelper.objectMapper` to a string
4. Compare string content with what you expect

**Comment by Riduidel** [comment]
Fixed in d7336cffc06395ca43479f1311c7d16e47142e5b
